src/scss/_objects.content-card.scss

```diff
@@ -0,0 +1,41 @@
+.content-card {
+  margin: halve($inuit-base-spacing-unit) 0;
+
+  > .box {
+    background-color: #fff;
```

Comment:
The Grommet core team would probably prefer that you use the proper variable here, in this case I think `$background-color` should work.